manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -336,6 +336,98 @@ Two parameters are required while the three others are optional.
                   field: categories
                   value: []
 
+Concatenate
+___________
+
+This action concatenates at least two values into a single value. A space separates each source value.
+
+The possible source attribute types are:
+ - text
+ - text area
+ - date
+ - identifier
+ - metric
+ - number
+ - price collection
+ - simple or multi select (option codes)
+
+The possible target attribute types are:
```

Comment:
target attributes ?

Reply:
nope it's the word "types" that must be pluralize not the attribute (it's just an adjective in this case)